scripts/pylib/twister/twisterlib/testplan.py

```diff
@@ -1261,12 +1249,14 @@ def verify_platforms_existence(self, platform_names_to_verify, log_info=""):
         as platform_allow or integration_platforms options) is correct. If not -
         log and raise error.
         """
+        _platforms = []
         for platform in platform_names_to_verify:
             if platform in self.platform_names:
-                continue
+                _platforms.append(self.get_platform(platform).name)
             else:
                 logger.error(f"{log_info} - unrecognized platform - {platform}")
                 sys.exit(2)
```

Comment:
> In case of unknown platforms in filters, raise an exception.

this commit summary is misleading: no exception, but just log and exit.

Reply:
yeah, it was an exception in the first try...